data/json/items/chemicals_and_resources.json

```diff
@@ -890,5 +890,24 @@
     "weight": 212,
     "volume": 1,
     "to_hit": -2
-    }
+    },{
+    "type" : "COMESTIBLE",
```

Comment:
> By the same logic why not edible gunpowder?

I think that's incorrect example. No one in right senses won't drink salt water, but it's still comestible.